### ISSUE TITLE
Some images are lost

### ISSUE THREAD

**Issue by pyrog** [kerel-fs/ogn-rdb#17]
Exemple [Rigi](https://ogn.peanutpod.de/#/receivers/Rigi) receiver.
Source: http://wiki.glidernet.org/list-of-receivers#toc26

Lost pictures:

- [Spectrogram (filter off)](http://www.flarm.com/files/FTS/Rx/rigi_unfiltered.jpg)
- [(filter on)](http://www.flarm.com/files/FTS/Rx/rigi_filtered.jpg)
- [Industrial PC](http://www.flarm.com/files/FTS/Rx/rigi_pc.JPG)

Parenthesis are not accepted in the regular expression, but in fact the image pattern is:
* start with "`[*`"
* url
* space "` `"
* description
* end with "`]`"

```python
photos_pattern = re.compile(r'\[\*?(?P<photo_url>[^\ \[\]]*)\ (?P<name>[^\ \[\]]*)\]')
image_url_pattern = re.compile(r".*\.(jpg|png|gif)$", re.IGNORECASE)
```

Wiki source code for the pictures section:
```wiki
[*/local--files/list-of-receivers/rigi1.jpg Panorama]
[*/local--files/list-of-receivers/rigi2.jpg Hotel]
[*http://www.flarm.com/files/FTS/Rx/rigi_combline_dvbt.JPG filter]
[*http://www.flarm.com/files/FTS/Rx/rigi_unfiltered.jpg Spectrogram (filter off)]
[*http://www.flarm.com/files/FTS/Rx/rigi_filtered.jpg (filter on)]
[*http://www.flarm.com/files/FTS/Rx/rigi_pc.JPG Industrial PC]
```

PS: I saw also some pictures with the `.jpeg` extension 😉

**Comment by pyrog** [comment]
More images lost:
http://openglidernetwork.wdfiles.com//local--files/list-of-receivers/SuttonBnk_Feb16.pngSuttonBnk
http://openglidernetwork.wdfiles.com//local--files/list-of-receivers/UKBSNbeta1.jpgphoto

```
[*/local--files/list-of-receivers/SuttonBnk_Feb16.png  SuttonBnk twds the South]
[*/local--files/list-of-receivers/UKBSNbeta1.jpg  photo (beta1 install)]
```

**Comment by pyrog** [comment]
Found image without extension:
[https://lh3.ggpht.com/-cRMdpzbv7YE/VNzoaxufTyI/…/s1536/photo](https://lh3.ggpht.com/-cRMdpzbv7YE/VNzoaxufTyI/AAAAAAAAAAo/hetfvz2Z45ALYUINkH1fMAzR16oMuqyVACJkC/s1536/photo)

Possible solutions:
- The easiest: add it manually (.jpg in this case).
- The cleanest: automatically browse the file and get it's real type.

**Comment by pyrog** [comment]
If file don't end by an image extension i.e. `.jpg` → cheat: add a fake `?.jpg` or `&.jpg` parameter 😉

This case happened for images published in blogs, Facebook…

`http://lh3.ggpht.com/-Pg7Lxe3CUok/VNJENIGdK8I/AAAAAAAAAAg/8YMgkOGpZusudQ-iWlZFWjfnNgc58C7OACJkC/s1536/photo`**&.jpg**

`http://scontent-cdg2-1.xx.fbcdn.net/v/t1.0-9/14317495_1020571118042209_1611534253014821319_n.jpg?oh=e0b46a6207d3a8aceb9af998ca77f2bc&oe=58ED0C8B`**&.jpg**

**Comment by snip** [comment]
I think that correct way to find file type is to look at http header and not to URL.

**Comment by kerel-fs** [comment]
No, I will not write/host a crawler to get the file type of some images. ;)
Imho we should focus on a replacement of the wiki list which helps the user/editors to enter correct urls.

**Comment by pyrog** [comment]
> Imho we should focus on a replacement of the wiki list

👍
